tests/Nhs.Appointments.Api.Integration/Scenarios/RoleManagement/RoleManagementFeatureSteps.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Nhs.Appointments.Api.Json;
+using Nhs.Appointments.Api.Models;
+using Xunit.Gherkin.Quick;
+
+namespace Nhs.Appointments.Api.Integration.Scenarios.RoleManagement;
+
+[FeatureFile("./Scenarios/RoleManagement/RoleManagement.feature")]
+public sealed class RoleManagementFeatureSteps : BaseFeatureSteps
+{
+    private  HttpResponseMessage _response;
+    private HttpStatusCode _statusCode;
+    private GetRolesResponse _actualResponse;
+    
+    [Given("There are existing roles")]
+    public Task RoleAssignments()
+    {
+        return Task.CompletedTask;
```

Comment:
Maybe a comment here to explain why this is blank and that we're relying on basic steup for the tests to provide the roles